jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -659,8 +662,45 @@ YUI.add('deployment-flow', function() {
         </div>);
     },
 
+    /**
+      Report whether going forward with the deployment is currently allowed.
+
+      @method _deploymentAllowed
+      @returns {Bool} Whether deployment is allowed.
+    */
+    _deploymentAllowed: function() {
```

Comment:
I find this syntax a little hard to read, what do you think about consolidating them into two blocks with a single return each?

```JavaScript
// Do not allow deploy when:
if (
  // No deployments are possible without a model name.
  !this.state.modelName ||
  // Check that we have a cloud where to deploy to.
  !this.state.cloud ||
  // Check that the user can deploy and they are not already deploying.
  this.props.acl.isReadOnly() || this.state.deploying) {
  return false;
}
// Allow deploy if:
...
```